packages/shared/dashboards/utilization-card/prometheus-utilization-item.tsx

```diff
@@ -49,6 +83,42 @@ export const PrometheusUtilizationItem: React.FC<PrometheusUtilizationItemProps>
     );
   };
 
+type TotalUtilizationItemProps = UtilizationItemProps & {
+  basePath: string;
+  defaultBasePath: string;
+  duration: number;
+  totalQuery: string;
+};
+
+const TotalUtilizationItem: React.FC<TotalUtilizationItemProps> = (props) => {
+  const [total, totalError, totalLoading] = useCustomPrometheusPoll({
+    query: props.totalQuery,
+    endpoint: 'api/v1/query_range' as any,
+    delay: props.duration,
```

Comment:
shouldn't this be "timespan" @bipuladh instead of "delay" ??
```suggestion
    timespan: props.duration,
```

Reply:
I can see "delay" being used in "PrometheusUtilizationItem" FC as well... but seems wrong to me...

Reply:
agreed

Reply:
Done.